.travis.yml

```diff
@@ -2,6 +2,10 @@ language: generic
 sudo: required
 dist: trusty
 
+branches:
```

Comment:
does this still run on pull requests?

Reply:
the "merge build" still runs (as you can see below), but the "branch build" does not, and I guess that's exactly what we want